requirements.txt

```diff
@@ -1,21 +1,17 @@
-# Pin these requirements to the latest versions when you create a new project,
-# e.g. Django==3.0.2.
-Django
-psycopg2-binary
-gunicorn
-dj-database-url
-whitenoise
-django-compressor
-sentry-sdk
-django-money
-django-datatables-view
+Django==3.1.7
```

Comment:
Can you confirm these requirements match those pinned in #82, which I generated from the production deployment, except where noted? https://github.com/fpdcc/ccfp-asset-dashboard/pull/82/files#r764218837

Reply:
Yeah I copy/pasted from your file. Just double checked and it looks right.

Reply:
Thanks!